docs/docs/dev_docs/tutorials/token_portal/main.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Token Portal Tutorial
```

Comment:
I would call it token bridge since it shows the whole thing (and not just what happens on L1). Bridge is also a more familiar term than "portal"